### PR TITLE
refactor!: move slider implementation from swiper to splidejs

### PR DESCRIPTION
**Pull request by nullxDEADBEEF**
# Description

Please include a summary of the change and which issue is fixed.

Changes the implementation of our sliders from `swiper` to `splidejs`

Fixes #

### CODE REVIEW

**Comment by AndreasPB** [comment]
Is it possible to have more than just one showing in seasons?

**Comment by nullxDEADBEEF** [comment]
> Is it possible to have more than just one showing in seasons?
> 
> 

Its quite difficult with small screen sizes, and I thought it looked a bit weird if there was another season half visible on the right

File: frontend/src/lib/components/details/seasons.svelte [hunk 1]
```diff
@@ -1,90 +1,82 @@
...
 </div>
+
+<style global>
+  @media (min-width: 320px) {
+    :global(.splide-seasons) {
+      display: flex;
+      justify-content: center;
+    }
+  }
+</style>
```
[the remark this answers is not in the document]
Comment (nullxDEADBEEF) [reply frontend/src/lib/components/details/seasons.svelte:82]: yeah this is for when the screen size is very small, we center the season

File: frontend/yarn.lock [hunk 2]
```diff
@@ -268,6 +268,18 @@
     estree-walker "^2.0.2"
     picomatch "^2.3.1"
 
+"@splidejs/splide@^4.1.3":
```
[the remark this answers is not in the document]
Comment (nullxDEADBEEF) [reply frontend/yarn.lock:271]: yeah you're right, just pushed an updated lockfile

**Review by AndreasPB — APPROVED** [review]

**Review by AndreasPB — COMMENTED** [review]
EDIT: nvm :D

**Review by AndreasPB — APPROVED** [review]

**Pull request merged**